python/az/aro/azext_aro/__init__.py

```diff
@@ -20,6 +20,17 @@ def __init__(self, cli_ctx=None):
                          custom_command_type=aro_custom)
 
     def load_command_table(self, args):
+        from azure.cli.core.aaz import load_aaz_command_table
```

Comment:
same as other comments. The import below in the `try` stays, I think.

Reply:
I put the `try` at the top as well, tested the ext and it works as it it.